chapter11/spg/exercise2.pl

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/perl
+
+use strict;
+use warnings;
+use Stereotypes qw( dow_to_name moy_to_name );
+
+my ( $sec, $min, $hour, $mday, $mon, $year, $wday ) = localtime;
+
+printf "Today is %s %s %s %s\n", dow_to_name( $wday ), moy_to_name( $mon ), $mday, $year += 1900;
```

Comment:
You probably should have used `%u` for the numbers,  and if you had used `Time::Piece` you could have avoided the crazy `+= 1900`